edns.go

```diff
@@ -128,8 +128,17 @@ func (rr *OPT) Do() bool {
 }
 
 // SetDo sets the DO (DNSSEC OK) bit.
-func (rr *OPT) SetDo() {
-	rr.Hdr.Ttl |= _DO
+func (rr *OPT) SetDo(do ...bool) {
+	// if we pass an argument, set the DO bit to that value
```

Comment:
Put this comment in the exported doc string above the function. Also tell we will silently ignore more than 1 argument.